homeassistant/components/resolution_center/websocket_api.py

```diff
@@ -60,3 +78,63 @@ def ws_dict(kv_pairs: list[tuple[Any, Any]]) -> dict[Any, Any]:
     ]
 
     connection.send_result(msg["id"], {"issues": issues})
+
+
+class ResolutionCenterFlowIndexView(FlowManagerIndexView):
+    """View to create issue fix flows."""
+
+    url = "/api/resolution_center/issues/fix"
+    name = "api:resolution_center:issues:fix"
+
+    @RequestDataValidator(
+        vol.Schema(
+            {
+                vol.Required("handler"): str,
+                vol.Required("issue_id"): str,
```

Comment:
Why isn't issue_id used as the handler ?

Reply:
Ah that's because the issue registry works like that 👍

Reply:
As discussed, we need the domain too